TestProject/output/GIFOutput.php

```diff
@@ -39,23 +39,22 @@ function outputBoard(Board $_board)
             }
         }
 
-        //Creates a new image which is going to be saved under -> TestProject/imageOutput/*.png
+        //Creates a new image in png-format
         imagepng($pngImage, "imageOutput/" . sprintf("image-%03d", $this->imageIndex) . ".png");
         $this->imageIndex++;
     }
 
     /**
-     * Creates a gif-file that is going to be saved under -> TestProject/imageOutput/*.gif
+     * A gif-file is automatically created by hand of the created images from the board.
      */
     function finishOutput()
     {
         $gif = new GifCreator();
 
-        //Throws the respective error-message for an incorrect process function process
+        //Throws the respective error-message for an incorrect function process
         try {
-            //Creates a gif of the created image-files under -> TestProject/imageOutput/*.png
+            //An animated gif is automatically created and saved under the respective repository.
             $gif->create(glob("imageOutput/*.png"), 50, 0);
-            //Saves the gif inside the respective directory
             file_put_contents("imageOutput/animation.gif", $gif->getGif());
```

Comment:
Please also delete all the temporary png images that the output created during its run. The user only wants a single gif as output, not hundreds of pngs also! :-)